cloudvolume/datasource/precomputed/image/common.py

```diff
@@ -68,18 +68,26 @@ def gridpoints(bbox, volume_bbox, chunk_size):
     yield Vec(x,y,z)
 
 def compressed_morton_code(gridpt, grid_size):
+  gridpt = np.asarray(gridpt, dtype=np.uint32)
```

Comment:
This is fine, but you might want to know about `lib.toiter(gridpt, return_multple=True)`